rasa/core/training/dsl.py

```diff
@@ -73,10 +73,10 @@ def add_checkpoint(self, name: Text, conditions: Optional[Dict[Text, Any]]) -> N
             self.start_checkpoints.append(Checkpoint(name, conditions))
         else:
             if conditions:
-                logger.warning(
+                warnings.warn(
                     "End or intermediate checkpoints "
```

Comment:
can this be one line?

Reply:
You have to take that one up with the formatter :P